getnacelle/contentful-app-nacelle
# chore: update forma to v4

## Why are these changes introduced?

Addresses [ENG-9419](https://nacelle.atlassian.net/browse/ENG-9419) 

## What is this pull request doing?

Updates forma to v4

## How to test

- Make sure node 16 is being used
- `npm run start` 
- Choose the Prairie wind apparel space with the `staging` environment and the app should be installed by default

## Checklist



[ENG-9419]: https://nacelle.atlassian.net/browse/ENG-9419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
